Sources/KlaviyoForms/InAppForms/IAFPresentationManager.swift

```diff
@@ -54,6 +54,13 @@ class IAFPresentationManager {
             return
         }
 
+        guard viewController == nil else {
+            if #available(iOS 14.0, *) {
+                Logger.webViewLogger.log("Webview is already set up; ignoring request.")
+            }
+            return
+        }
+
```

Comment:
Is there any in-between time that needs to be accounted for, where a webview has been created, but the `viewController` is not?